ember-toucan-core/src/components/form/textarea-field.hbs

```diff
@@ -0,0 +1,28 @@
+<div class="flex flex-col">
+  <Form::Field as |field|>
+    <field.Label for={{field.id}} data-textarea-label>{{@label}}</field.Label>
+
+    {{#if @hint}}
+      <field.Hint id={{field.hintId}} data-textarea-hint>{{@hint}}</field.Hint>
+    {{/if}}
+
+    <field.Control>
+      <textarea
+        aria-invalid={{if @error "true"}}
+        aria-describedby="{{if @error field.errorId}} {{if @hint field.hintId}}"
+        class="block h-20 min-h-6 p-1 bg-overlay-1 focus:outline-none rounded-sm shadow-focusable-outline focus:shadow-focus-outline
+          {{if @isDisabled 'text-disabled' 'text-titles-and-attributes'}}"
+        disabled={{@isDisabled}}
+        id={{field.id}}
+        ...attributes
```

Comment:
Ah! Good call.  I was initially thinking the consumers would control all of that, but now I remember controlled vs uncontrolled components in React and it seems like we should definitely set `value` for input, textarea, etc. and have an internal `setValue` like your link and this https://github.com/CrowdStrike/ember-headless-form/blob/main/packages/ember-headless-form/src/-private/components/control/textarea.ts#L43 right?

Reply:
I should do this for InputField also.

Reply:
Yeah, right. `@value` is the one thing, but `@onChange` (or however you want to name it) is also important. First I thought users could just apply `{{on ...}}` themselves, but indeed the action you linked to shows an important point: it would be way more convenient to not have to deal with getting the (changed) value from `event.target`!